main/demo_tasks/ota_over_mqtt_demo/ota_over_mqtt_demo.c

```diff
@@ -1037,12 +1037,11 @@ static bool sendSuccessMessage( void )
          * AWS IoT Jobs library:
          * Creating the message which contains the status of OTA job.
          * It will be published on the topic created in the previous step.
-         */
-        size_t messageBufferLength = Jobs_UpdateMsg( Succeeded,
-                                                     "2",
-                                                     1U,
-                                                     messageBuffer,
-                                                     UPDATE_JOB_MSG_LENGTH );
+                */
```

Comment:
```suggestion
         */
```
I guess this might be unintentionally modified